src/components/Breadcrumb/BreadcrumbItem.tsx

```diff
@@ -1,24 +1,29 @@
+import { Link, LinkElement } from "./Breadcrumb"
 import styles from "./Breadcrumb.module.css"
 
-export interface LinkProps {
-  href: string
-  label: string
-  isCurrent?: boolean
+export interface BreadcrumbItemProps {
+  isCurrent: boolean
+  link: Link
+  linkElement?: LinkElement
 }
 
-export const BreadcrumbItem = ({ href, isCurrent, label }: LinkProps) => {
+export const BreadcrumbItem = ({
+  isCurrent,
+  link,
+  linkElement,
+}: BreadcrumbItemProps) => {
   const ariaCurrent = isCurrent ? "page" : undefined
 
   return (
-    <li className={styles.BreadcrumbItem}>
-      <a href={href} className={styles.link} aria-current={ariaCurrent}>
-        {label}
-      </a>
-      {!isCurrent && (
-        <span aria-hidden="true" className={styles.Separator}>
-          /
-        </span>
+    <li className={styles.list}>
+      {linkElement?.renderItem ? (
+        linkElement.renderItem({ item: link, isCurrent })
+      ) : (
+        <a href={link.href} aria-current={ariaCurrent}>
```

Comment:
💭 I wonder if it would be at all useful to pass down the class for use on a passed item? Maybe a bad idea. 🤷 
```suggestion
      {linkElement?.renderItem ? (
        linkElement.renderItem({ item: link, isCurrent, className: styles.link })
      ) : (
        <a href={link.href} aria-current={ariaCurrent} className={styles.link}>
```

Reply:
@roblafeve I think this should help simply the css too

Reply:
Ah, yeah—explicitly styling by class vs nested element... yeah, I think so